### PR TITLE
LPS-201191 Add Object Definitions test cases to be exported in CSV

### PR DESCRIPTION
**Pull request by sergiojimcos**
**What is new?**
- Create some tests cases to export Object Definition in a default company and in a new one.
- Add new CSV file to compare with the exported one.

**JIRA Related Ticket**
https://liferay.atlassian.net/browse/LPS-201191

### CODE REVIEW

**Comment by sergiojimcos** [comment]
ci:test:relevant

**Review by 4lejandrito — CHANGES_REQUESTED** [review]
Look what I've done with these 2 tests:

```java
       @Test
	public void testExportObjectDefinition() throws Exception {
		_setUpObjectDefinition("TestObjectJSON1");

		_objectMapper.setFilterProvider(
			new SimpleFilterProvider() {
				{
					addFilter(
						"Liferay.Vulcan",
						VulcanPropertyFilter.of(
							new HashSet<>(_objectDefinitionExportFieldNames),
							null));
				}
			});

		_assertEqualsExport(
			_getExpectedJsonNode(_objectDefinition1),
			_objectDefinitionExportFieldNames,
			_getActualJsonNode(
				_objectMapper.readTree(
					_getZipInputStream(
						_getObjectDefinitionExportInputStream(
							BatchPlannerPlanConstants.EXTERNAL_TYPE_JSON,
							_objectDefinitionExportFieldNames))),
				_objectDefinition1.getShortName()));
	}

	@Test
	public void testExportObjectDefinitionCSV() throws Exception {
		_setUpObjectDefinition("TestObjectCSV");

		_assertEqualsExportCSV(
			_getObjectDefinitionExportInputStream(
				BatchPlannerPlanConstants.EXTERNAL_TYPE_CSV,
				_objectDefinitionExportCSVFieldNames),
			_getInputStream("csv/expected-object-definition.csv"),
			_objectDefinition1.getShortName());
	}
```
Do the same with the others if possible.

File: modules/apps/batch-planner/batch-planner-test/src/testIntegration/java/com/liferay/batch/planner/batch/engine/broker/test/BatchEngineBrokerTest.java [hunk 1]
```diff
@@ -717,6 +717,36 @@ private void _assertEqualsExport(
...
+				new InputStreamReader(expectedInputStream));
+
+		_assertColumnNames(
+			_getContentRow(actualUnsyncBufferedReader.readLine()),
+			_getContentRow(expectedUnsyncBufferedReader.readLine()));
+
+		String actualLineString = actualUnsyncBufferedReader.readLine();
+
+		while (actualLineString != null) {
+			if (actualLineString.contains(objectDefinitionName)) {
```
Comment (4lejandrito) [inline modules/apps/batch-planner/batch-planner-test/src/testIntegration/java/com/liferay/batch/planner/batch/engine/broker/test/BatchEngineBrokerTest.java:740]: If this if is never true, the test will pass, and it should fail.

**Review by 4lejandrito — CHANGES_REQUESTED** [review]

**Comment by sergiojimcos** [comment]
ci:test:sf

**Review by 4lejandrito — APPROVED** [review]

**Comment by Hijky** [comment]
The failures are not related to your PR, it's okay to forward it.

**Comment by sergiojimcos** [comment]
Sent to Brian here: https://github.com/brianchandotcom/liferay-portal/pull/143910

**Pull request closed without merging**